src/featurecounts/script.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+if [[ $par_tmpdir ]] && [[ ! -d "$par_tmpdir" ]]; then
+    mkdir -p $tmpDir
+fi
+
+if [[ $par_r_path ]] && [[ ! -d "$par_r_path" ]]; then
+    mkdir -p $par_r_path
```

Comment:
Don't forget to quote paths in case the user adds spaces to the directory name

```suggestion
    mkdir -p "$par_r_path"
```

Also, I suggested to rename this argument to `--results_path`, so:


```suggestion
    mkdir -p "$par_results_path"
```